packages/Table/tests/cypress/Table.cypress.js

```diff
@@ -0,0 +1,154 @@
+import { getStoryUrlPrefix } from "../../../../.storybook/storyTree";
+
+describe("Table", () => {
+  it("Should render the table data", () => {
+    cy.visitStorybook(`${getStoryUrlPrefix("Table")}--basic`);
+
+    cy.getByText("Name").should("be.visible");
+    cy.getByText("LastName").should("be.visible");
+
+    cy.getByText("Charles").should("be.visible");
+    cy.getByText("Claude").should("be.visible");
+    cy.getByText("Alan").should("be.visible");
+    cy.getByText("John von").should("be.visible");
+    cy.getByText("William").should("be.visible");
+    cy.getByText("Douglas").should("be.visible");
+    cy.getByText("Robert").should("be.visible");
+    cy.getByText("Steve").should("be.visible");
+    cy.getByText("Grace Murray").should("be.visible");
+    cy.getByText("Vint").should("be.visible");
+
+    cy.getByText("Babbage").should("be.visible");
+    cy.getByText("Shannon").should("be.visible");
+    cy.getByText("Turing").should("be.visible");
+    cy.getByText("Neumann").should("be.visible");
+    cy.getByText("Shockley").should("be.visible");
+    cy.getByText("Engelbart").should("be.visible");
+    cy.getByText("Noyce").should("be.visible");
+    cy.getByText("Wozniak").should("be.visible");
+    cy.getByText("Hopper").should("be.visible");
+    cy.getByText("Cerf").should("be.visible");
+
+    cy.getByText("Name")
+      .should("have.css", "font-weight")
+      .should("eq", "700");
+    cy.getByText("LastName")
+      .should("have.css", "font-weight")
+      .should("eq", "700");
+  });
+
+  it("Should display zebrastripes", () => {
+    cy.visitStorybook(`${getStoryUrlPrefix("Table")}--has-zebra-stripes`);
+
+    cy.getAllByRole("row")
+      .eq(2)
+      .should("have.css", "background-color")
+      .should("eq", "rgb(247, 247, 247)");
```

Comment:
I think if you change "rgb(247, 247, 247)" for `const grey = ""rgb(247, 247, 247)"` might be easier to understand same goes for `const white = "rgb(255, 255, 255)"`